docs/tutorials/quickstart.md

```diff
@@ -1,8 +1,194 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.15.2
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
 # Quickstart
 
-```{note}
-This is a placeholder for a section in the new {ref}`jax-tutorials`.
+**JAX is a library for array-oriented numerical computation (*a la* [NumPy](https://numpy.org/)), with automatic differentiation and JIT compilation to enable high-performance machine learning research**.
```

Comment:
Nit: Use more accessible language for readers who don't know this French expression:

```suggestion
**JAX is a library for array-oriented numerical computation (like [NumPy](https://numpy.org/)), with automatic differentiation and JIT compilation to enable high-performance machine learning research**.
```